src/raven.js

```diff
@@ -413,6 +413,16 @@ Raven.prototype = {
             timestamp: now() / 1000
         }, obj);
 
+        if (isFunction(this._globalOptions.breadcrumbCallback)) {
+            var result = this._globalOptions.breadcrumbCallback(crumb);
+
+            if (result) {
+                crumb = result;
+            } else {
```

Comment:
Perhaps another related question is that if we don't want to throw, that undefined would cause it to use the original data object. What should happen if the callback returns a string? What does `dataCallback` do?